data retrieval/src/main/java/de/tinf15b4/kino/retrieval/tmdb/TmdbDataRetriever.java

```diff
@@ -48,19 +46,20 @@ public Movie getMovie(Movie movie) throws MovieDbException {
         List<MovieInfo> movies = searchInstance.searchMovie(movie.getName(), 1, "de-DE", false, 0, 0, SearchType.NGRAM)
                 .getResults();
 
-        if (movies.size() != 0) {
+        if (!movies.isEmpty()) {
             // reorder results by year
             // We want the movie from the most recent year (e.g. the 2017 version of "Die Schöne und das Biest"),
             // but for movies from the same year, we want the movie db to decide ("Alien: Covenant" vs the Rick and Morty clip)
             movies.sort(Comparator.comparing((MovieInfo m) -> m.getReleaseDate().substring(0,Math.min(m.getReleaseDate().length(), 4))).reversed());
 
             MovieInfo mi = moviesInstance.getMovieInfo(movies.get(0).getId(), "de-DE", "");
-            movie = new Movie();
```

Comment:
afaik we only aspect the method to return a movie. the change was the result of sonarqube complaining a code smell